packages/backend/src/peripherals/starkware/L2TransactionDownloader.ts

```diff
@@ -0,0 +1,56 @@
+import { Logger } from '../../tools/Logger'
+import { L2TransactionRepository } from '../database/L2TransactionRepository'
+import { StateUpdateRecord } from '../database/StateUpdateRepository'
+import { FeederGatewayClient } from './FeederGatewayClient'
+
+export class L2TransactionDownloader {
```

Comment:
I don't see how it's any different than all other collectors we have.

I suggest renaming to FeederGatewayCollector, renaming "sync" to "collect" and moving to "/collectors" folder.